# Request
float64 gripper_position

---

#Response
bool success